i-slide.js

```diff
@@ -226,10 +226,36 @@ class ISlide extends HTMLElement {
 
     const width = parseInt(this.getAttribute('width') ?? 300, 10);
 
+    // Retrieve styles to be applied to :host
+    function getHostStyles(height) {
+      const heightProp = height ? `
+        height: ${height}px;
+      ` : '';
+
+      return `
+        :host {
+          display: block;
+          position: relative;
```

Comment:
does this intersect with a situation where one were to try use an `<i-slide>` in a `position: absolute` setting?

Reply:
That's a good question, and the answer seems to be yes. What bugged me is that these properties are not directly visible on the element in the developer console, but properties applied to `:host` are indeed applied to the `<i-slide>` element as seen by the containing document, and can thus be overwritten by styling the element.

That is not what we need. We need to make sure that the content in the shadow root never overflows, regardless of styles applied to the element in the containing document (we'll also want to apply the right scale at any time, but that's a separate issue)

I think that the right way to achieve that is to apply the `position:relative` and `overflow:hidden` at the root element in the shadow DOM. At least that seems to work when I test things locally. I'll update the PR accordingly.